spring-cloud-starter-single-step-batch-job/src/main/java/org/springframework/cloud/task/batch/autoconfigure/rabbit/AmqpItemReaderProperties.java

```diff
@@ -28,8 +28,15 @@
 @ConfigurationProperties(prefix = "spring.batch.job.amqpitemreader")
 public class AmqpItemReaderProperties {
 
+	/**
+	 * Enables or disables the AmqpItemReader.  Defaults to false.
```

Comment:
Make "AmqpItemReader" be code: {@code AmqpItemReader}
Make "false" be code: {@code false}
Change the double space after the first period to a single space.